README.md

```diff
@@ -376,6 +376,13 @@ Additional configuration can be passed via the following environment variables:
 * EXPORT_METRICS_INTERVAL_SECONDS - `default 20` interval to report lag/offset/end to metricsRegistry
 * CLOUDWATCH_ON - `default false` reports metricsRegistry to cloudwatch, TO_REGISTRY will need to be switched on!
 * CLOUDWATCH_NAME - `default 'remora'` name to appear on cloudwatch
+* DATADOG_ON - `default false` reports metricsRegistry to Datadog, TO_REGISTRY will need to be switched on!
```

Comment:
I have no issue if you want to give DataDog it's own Header, i think I should do the same with Cloudwatch considering the number of integrations with metrics tools are growing.

e.g. ## Configuring Remora with Data Dog

Reply:
I did this separation, and also added some more information about the integration with the datadog agent